tests/planner/ompl/test_MotionValidator.cpp

```diff
@@ -29,7 +29,7 @@ class MotionValidatorTest : public ::testing::Test
         std::make_shared<MockTranslationalRobotConstraint>(
             stateSpace, Eigen::Vector3d(-0.1, -0.1, -0.1),
             Eigen::Vector3d(0.1, 0.1, 0.1));
-    auto vchecker =
+    ::ompl::base::StateValidityCheckerPtr vchecker =
```

Comment:
@jslee02 not specifying explicitly the type was resulting in a build error due to ambiguity of the variable type, between `StateValidityCheckerPtr` and `StateValidityCheckerFn`. I thought you might want to confirm this change.